src/index.html

```diff
@@ -1,13 +1,65 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
+
+
+    <link rel="stylesheet" href="style.css">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <div class="product-card__product-img">
+        <img
+          src="./images/imac.jpeg"
+          alt="product_image"
+          class="product-card__mac-image"
+          data-qa="hover"
+          >
+      </div>
+      <div class="product-card__main-info">
+        <div class="product-card__product-title">
+          <h6 class="product-card__product-name">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h6>
+
+          <p class="product-card__product-code">
+              Product code: 195434
+          </p>
+        </div>
+
+        <div class="product-card__product-statistics">
+          <div class="product-card__product-stars">
+            <div class="product-card__stars star"></div>
+            <div class="product-card__stars star"></div>
+            <div class="product-card__stars star"></div>
+            <div class="product-card__stars star"></div>
+            <div class="product-card__stars star"></div>
+          </div>
+          <div class="product-card__review">
+            <p class="review">Reviews: 5</p>
+          </div>
+        </div>
+
+        <div class="product-card__product-price">
+          <p class="product-card__price-name">
+              Price:
+          </p>
+          <p class="product-card__price-cost">
+            <strong>$2,199</strong>
+          </p>
+        </div>
+      </div>
+      <div class="product-card__buy-button">
+        <a href="#" data-qa="hover"><button class="product-card__btn">buy</button></a>
```

Comment:
```suggestion
        <a href="#" data-qa="hover">
          <button class="product-card__btn">buy</button>
        </a>
```

Reply:
use one tag per line.
But in this case, you can use only `a`. No need to put `button `inside it